harvard-lts/librarycloud_ingest
# Ltscloud 1125

**XSL xform, security patch, file logging, streamline docker**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/LTSMAINT-592)

# What does this Pull Request do?
- XSLT changes for bad gov doc flag (LTSMAINT-612) and concatenation issue (LTSMAINT-613)
- Jackson bind  version bump fixes high severity dependabot (LTSMAINT-615)
- Revert to file-based log4j (because of ELK issues); read in props rather than checked in (LSTMAINT-614)
- Streamline Dockerfile, use docker-compose, and use lcadm user (LTSMAINT-621)

# How should this be tested?
- Check out branch and run > docker compose -f docker-compose.yml build; there should be no errors
- Push to registry and bounce container on qa (done successfully, and tested ingest)
